src/Adapter/Requests/Models/Bucket.cs

```diff
@@ -95,6 +100,10 @@ public class EntityConfig : IEntityTypeConfiguration<Bucket>
             /// <inheritdoc />
             public void Configure(EntityTypeBuilder<Bucket> builder)
             {
+                builder
+                .HasIndex(bucket => bucket.RemoteId)
+                .IsUnique();
```

Comment:
This should be compositely unique with major parameters

Reply:
Compositely unique key with major parameters would be too big at the moment, but perhaps can improve the way major parameters are stored in the future